app/models/collecting_event.rb

```diff
@@ -820,15 +822,18 @@ def containing_geographic_items
       #
       #  Struck EGI, EGI must contain GI, therefor anything that contains EGI contains GI, threfor containing GI will always be the bigger set
       #   !! and there was no tests broken
-      # GeographicItem.are_contained_in_item('any_poly', self.geographic_items.to_a).pluck(:id).uniq
-      gi_list = GeographicItem.containing(*geographic_items.pluck(:id)).pluck(:id).uniq
+      # GeographicItem.st_covers_item('any_poly', self.geographic_items.to_a).pluck(:id).uniq
+      gi_list = GeographicItem
+       .covering_union_of(*geographic_items.pluck(:id))
+       .pluck(:id)
+       .uniq
 
     else
       # use geographic_area only if there are no GIs or EGIs
       unless self.geographic_area.nil?
         # unless self.geographic_area.geographic_items.empty?
         # we need to use the geographic_area directly
-        gi_list = GeographicItem.are_contained_in_item('any_poly', self.geographic_area.geographic_items).pluck(:id).uniq
+        gi_list = GeographicItem.st_covers_item('any_poly', self.geographic_area.geographic_items).pluck(:id).uniq
```

Comment:
Let's get rid of the postfixed `_item`, and see how that feels.  Then we could assume that `st_` class methods all are named as defined in postgis.